src/schema/ghga.yaml

```diff
@@ -2469,10 +2471,22 @@ enums:
     description: >-
       Enum to capture whether a Sample in a Study is to be considered as Case or Control.
     permissible_values:
-      control:
-        description: The Sample is to be treated as Control
-      case:
-        description: The Sample is to be treated as Case
+      unable_to_assess_case_or_control_status:
+        description: Unknown
+      neither_case_or_control_status:
+        description: The participant is neither a true case or true control for the phenotype under consideration
+      probable_case_status:
+        description: The participant is a probable case for the phenotype under consideration
+      probable_control_status:
+        description: The participant is a probable control for the phenotype under consideration
+      true_case_status:
+        description: The participant is a true case for the phenotype under consideration
+      true_control_status:
+        description: The participant is a true control for the phenotype under consideration
+      tumor:
```

Comment:
Where from do we have the values "tumor" and "healthy". They are not part of ontology http://purl.obolibrary.org/obo/NCIT_C99268 and they are also not listed in the permissible values in Control Vocabulary list.
This is a version of spreadsheet I am using: https://docs.google.com/spreadsheets/d/1VMli7kpmzyx7dq1v2vCSn_hcbFAU4vkcz76JG6kop4w/edit#gid=862355172

Reply:
We added those fields to capture Tuebingen data.